csunplugged/at_home/content/en/unlocking-the-secret-in-product-codes/activity-extra-information.md

```diff
@@ -0,0 +1,3 @@
+Note that you can make this calculation a bit easier by keeping only the ones digits after each calculation (this is called modulo 10 arithmetic). For example, 8 + 4 = 12, but you just keep the 2; or 9 x 3 = 27, and you can just keep the 7. It will give the same result if you discard the 10s digit, and with practice, you may even be able to do the calculation in your head!
```

Comment:
```suggestion
Note that you can make this calculation a bit easier by keeping only the ones digits after each calculation (this is called modulo 10 arithmetic).
For example, 8 + 4 = 12, but you just keep the 2; or 9 x 3 = 27, and you can just keep the 7.
It will give the same result if you discard the 10s digit, and with practice, you may even be able to do the calculation in your head!
```